README.md

```diff
@@ -461,6 +461,23 @@ Ran 4 tests in 0.153s
 OK
 ```
 
+## Running tests
+
+You must rebuild C files for the tests to pick up your changes.  Try this for iterating:
+
+```
+$ python setup.py build_ext --inplace && python setup.py test
```

Comment:
This _is_ much faster than removing and rebuilding, so I think it still has a place for testing